### PR TITLE
update all.json

### PR DESCRIPTION
**Pull request by ical10**
|Site|Urlscan|Screenshot|
|--|--|--|
|bit-vestnetwork.com|https://urlscan.io/result/d67cc036-418b-4c49-a866-c4a5578c9a2e/dom/|https://user-images.githubusercontent.com/14356826/173289073-eba477ef-22e3-4cde-8785-70c9dc11be72.png|
|configdevminter.online|https://urlscan.io/result/a62612b5-476c-4b51-ad7b-d7c228a0b528/dom/|https://user-images.githubusercontent.com/14356826/173288151-31ed386e-b6aa-4b6b-898f-a9d33eca0254.png|
|connect-wallet-recovery.annelouisekrogh.dk|https://urlscan.io/result/7e4704c5-a383-4de6-b785-b9a62d0b3487/|https://user-images.githubusercontent.com/14356826/173288998-099ada3d-caac-4c89-9db1-6d3c680f0ec3.png|
|drasper27.github.io|https://urlscan.io/result/adc04869-0099-420f-afa4-f70f61901ee3/dom/|https://user-images.githubusercontent.com/14356826/173287960-48479db0-1ac0-4beb-82fd-8c88ff74e284.png|
|easywebsyncrination.com|https://urlscan.io/result/b7a441fa-710e-4883-9de0-483413e8e2dd/dom/|https://user-images.githubusercontent.com/14356826/173287979-52889ddc-7d65-4d0e-9192-0d69c4c317de.png|
|fixingprotocol.com|https://urlscan.io/result/b131b6c4-c1c4-4888-841b-77dccd14b714/dom/|https://user-images.githubusercontent.com/14356826/173289138-6af67bd5-0d0b-4850-a3b7-39ae7c84d3b8.png|
|itsmy-asset.com|https://urlscan.io/result/583932be-eb69-4a4a-b831-a3b519823300/dom/|https://user-images.githubusercontent.com/14356826/173288236-ff4a29a4-f213-4fda-a9f6-274bf8f8f3eb.png|
|mainbscrectify.com|https://urlscan.io/result/62b2fa7f-909d-42b0-9b68-ab900105686a/dom/|https://user-images.githubusercontent.com/14356826/173289174-474da341-2027-43f5-b8d4-2e974d69eae0.png|
|mainsync.net|https://urlscan.io/result/41985be8-bd24-47c5-89cb-6d6429181587/dom/|https://user-images.githubusercontent.com/14356826/173289011-bdd1240e-189f-4d61-97d5-d2bc8967892d.png|
|micropool-defi.co|https://urlscan.io/result/e3d69111-988e-4d95-bd7f-ca01b8cc248a/dom/|https://user-images.githubusercontent.com/14356826/173288069-5f5cc4f1-3fd5-40ba-905c-770126ab6c9c.png|
|securetokenrestore.com|https://urlscan.io/result/83c93f22-6b35-42b4-a825-3143b5eb00ad/|https://user-images.githubusercontent.com/14356826/173289422-5f9cce9e-1ac7-4659-8f77-65f3dbbadab9.png|
|smartcointechsolutiion.xyz|https://urlscan.io/result/2c6bbe21-efe9-4a39-ac24-dc153bfb2fca/|https://user-images.githubusercontent.com/14356826/173289473-116aa06c-b1d6-41d3-bab1-e4bc164a0f00.png|
|smartcointechsolution.art|https://urlscan.io/result/71ac40ae-86da-474d-9182-cca31c32cccc/|https://user-images.githubusercontent.com/14356826/173289648-2c706e4e-95bb-4012-89de-806bc767f11e.png|
|smartcointechsolution.live|https://urlscan.io/result/c1ee2bad-c2c6-499d-ac98-3880db744794/|https://user-images.githubusercontent.com/14356826/173289444-558eac0c-0235-4949-a0a0-4b64d65e925c.png|
|smartdappdata.io|https://urlscan.io/result/2375127a-3d5a-424d-b19c-34d32acc9465/dom/|https://user-images.githubusercontent.com/14356826/173289703-d83292b8-c093-4e7b-bfd4-2e2e81cd65eb.png|
|smartdappsauthorizatoin.live|https://urlscan.io/result/9e4489fe-d861-4cb6-b797-133c861ec684/dom/|https://user-images.githubusercontent.com/14356826/173289748-d973f633-800a-4959-9700-043645f99103.png|
|terra-supports.com|https://urlscan.io/result/906f94bd-e697-4b20-8bfa-4c7bbc00ee05/|https://user-images.githubusercontent.com/14356826/173288971-45bda7c7-208c-4b24-87ce-293e40291cfe.png|
|thebitkisa.live|https://urlscan.io/result/4f96b194-65b9-4143-b566-dcede387039c/dom/|https://user-images.githubusercontent.com/14356826/173288015-3a96e7c3-6629-48cc-832e-e86fe7fd634a.png|

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -1,6 +1,25 @@
 {
 	"allow": [],
 	"deny": [
+		"test1.consultando.eu",
```
Comment (jacogr) [inline all.json:4]: Seems to be a WP login?

File: all.json [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+		"smartdappdata.io",
+		"smartcointechsolution.art",
+		"smartdappsauthorizatoin.live",
+		"smartcointechsolutiion.xyz",
+		"smartcointechsolution.live",
+		"itsmy-asset.com",
+		"configdevminter.online",
+		"mainbscrectify.com",
+		"micropool-defi.co",
+		"drasper27.github.io",
```
Comment (jacogr) [inline all.json:21]: Taken down by GH.

Based on history, seems quite old - https://github.com/drasper27

File: all.json (hunk 1 above)
Comment (ical10) [reply all.json:4]: Yeah, but the full path redirects to malicious content. Gonna remove it.

File: all.json (hunk 2 above)
Comment (ical10) [reply all.json:21]: Still live when opened in different browser. Better to block it anyway, no?

Comment (ical10) [reply all.json:20]: Ah I get it. The content is still live though. Check out the urlscan + screenshots. It's still fresh. @jacogr

Comment (jacogr) [reply all.json:20]: Weird, went to the URL directly, it is down. No issues, doesn't hurt having it.

What is perplexing is that it stayed up so long, the repos on his account `live1`/etc is quite old. He/she probably just kept it in the bag and surfaced it recently.

**Review by jacogr — APPROVED** [review]

File: all.json (hunk 2 above)
Comment (ical10) [reply all.json:20]: Yeah, it just recently circulating on several discord servers. Really risky to let it be in the wild. Thanks for reviewing!

**Pull request merged**